_includes/extensions/writing-repeater.html

```diff
@@ -27,158 +47,131 @@ <h2 id="writing-extensions-repeater">Repeater Views</h2>
+				<a href="#writing-extensions-repeater-itemObject">item object</a>, or <code>false</code> if no action is desired.
 			</td>
 		</tr>
 		<tr>
-			<td>nested</td>
-			<td>array</td>
-			<td>Array of renderer objects. These objects use the same structure as the one currently being described.
-				Each object will run in order following the <code>render</code> function of the current object unless
-				otherwise specified.
+			<td>repeat</td>
+			<td>string</td>
+			<td>This string value specifies which array the <code>renderItem</code> function iterates over. It represents
+				an attribute of an object using dot notation. The root level value must use <code>'data'</code> or
+				<code>'this'</code>. If the root value uses <code>'data'</code>, the root object will be the <code>data</code>
+				object returned from the <code>dataSource</code>. If the root value uses <code>'this'</code>, the root object
+				will be the current repeater instance context of <code>this</code>. You can specify further levels separated
+				by dots. The final level must include an array or iteration will not function correctly. This array will
+				be passed as the <code>subset</code> attribute in the <code>helpers</code> object of the <code>renderItem</code>
+				function. The default value is <code>'data.items'</code>
 			</td>
 		</tr>
 		<tr>
-			<td>render</td>
+			<td>after</td>
 			<td>function</td>
-			<td>Runs after the <code>before</code> function. This function renders data
-				and adds content to the parent container. Provided a <a href="#writing-extensions-repeater-helpers">helpers</a>
-				object and <code>callback</code> function. The <code>callback</code> function accepts an object
-				with several attributes:
-				<ul>
-					<li><code>item</code></li>
-					<li><code>action</code></li>
-					<li><code>skipNested</code></li>
-				</ul>
-				The <code>item</code> attribute represents the desired content to be injected into the DOM as
-				either a <code>string</code> or <a href="http://learn.jquery.com/using-jquery-core/jquery-object/">jQuery object</a>.
-				Item content can include special <a href="#writing-extensions-repeater-dataAttr">data-attributes</a>
-				for additional rendering behaviors. The <code>action</code> attribute (string)
-				specifies how the <code>item</code> will inject in relation to the parent container. This value defaults
-				to <code>append</code>, but it can use any valid jQuery DOM injection method, such as 'prepend' or 'after').
-				The <code>skipNested</code> attribute, if set to <code>true</code>, will tell the repeater to
-				ignore the <code>nested</code> array for the current renderer.
+			<td>This function is called after the renderItems function, if no <code>render</code> function has been defined.
+				Provided a <code>helpers</code> object containing the parent <code>container</code> (the repeater canvas)
```

Comment:
The default container value switches at the `before`, `renderItems`, and `after` context. Had a bit of trouble explaining that... for `before` and `after`, container will be the repeater canvas. For `renderItems`, it will be the repeater canvas or a child node with `data-container="true"` on it, if found.